### PR TITLE
#4 Added plants vs pollution

### PR DESCRIPTION
**Pull request by Jxshvh**
Running the website client-side did not function so I was not able to test it, but I maintained the same structure as your other game so it should work.

### CODE REVIEW

**Comment by EdgeKing810** [comment]
Excellent! Just a couple of things before I merge.

* Changelog should be omitted. 
* Make a new directory called ``img/p-vs-p``, put the image there instead and correctly reference it in the file!

**Comment by Jxshvh** [comment]
Sorry for the late reply, I did in fact add the image and relevant folder: https://github.com/Jxshvh/kinesisdownloads.github.io/tree/gh-pages/img/plants-vs-pollution
I haven't been able to find the changelog of Plants vs Pollution.

File: _posts/2019-10-02-Plants-vs-Pollution.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout:     post
+title:      Plants vs Pollution
+date:       2019-10-02 00:00:00
+summary:    After humans have been polluting Earth with their factories, a nuclear incident causes plants to stand up, take things into their leaves and fight pollution.
+status:     Active, Open Beta
```
Comment (EdgeKing810) [inline _posts/2019-10-02-Plants-vs-Pollution.md]: Change ``status`` to ``Stalled``

File: _posts/2019-10-02-Plants-vs-Pollution.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+thumbnail:  
+---
+
+# Description
+
+The game will feature a lot of levels with progressively difficult aims. Groups of levels will take place in specific types of locations with unique plants. Among these locations, there'll be Forests, Deserts and Marshes. Levels will have Factory Slot(s) with of course, Factories, aimed towards producing Carbon Dioxide (CO2). Different Plants will be worth different amounts of CO2 and once their requirements satisfied, they can be placed in Empty Plant Slots to produce mainly Oxygen (O2). A Remove Plant button will also be present to assist the player in making Plant Slots empty. 
+
+# Changelog
```
Comment (EdgeKing810) [inline _posts/2019-10-02-Plants-vs-Pollution.md]: Remove the ``Changelog`` part.

File: _posts/2019-10-02-Plants-vs-Pollution.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+{% highlight html %}
+Version: 2.0.1
+{% endhighlight %}
+
+[Get it on Google Play!](https://play.google.com/store/apps/details?id=com.KinesisGames.PlantsvsPollution)
+
+Send us any [feedback](mailto:feedback@kinesisgames.net) you have!
+
+![](/img/prototypes/plants-vs-pollution.png)
```
Comment (EdgeKing810) [inline _posts/2019-10-02-Plants-vs-Pollution.md]: If you put the image for ``plants-vs-pollution`` in a different folder as I told you, reference it correctly here.

**Comment by Jxshvh** [comment]
The image folder and path were already correct, changelog has been removed

**Comment by EdgeKing810** [comment]
Alright!

**Pull request merged**